src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardController.js

```diff
@@ -51,51 +51,51 @@
         if (task === 'cancel' || task === 'backToTemplates') {
```

Comment:
We can also remove the 'backToTemplates' bit since we removed the backToTemplates button.